### PR TITLE
More mac changes to address compile issues

### PR DESCRIPTION
**Pull request by DanHeidinga**
See issue: #36

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Jenkins copyright check

File: runtime/ddrext/ddr.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ extern "C" {
 #include "windows.h"
 #endif
 
-#if defined(LINUX) || defined(AIXPPC)
+#if defined(LINUX) || defined(AIXPPC) || OSX
```
Comment (gacholio) [inline runtime/ddrext/ddr.h]: Missing defined.

Comment (gacholio) [reply runtime/ddrext/ddr.h]: Also copyrights.

**Comment by gacholio** [comment]
CI build is failing for reasons unrelated to this PR.  Testing doesn't cover OSX, so no need for that either.

**Pull request merged**